bot/exts/evergreen/splitify.py

```diff
@@ -0,0 +1,208 @@
+import asyncio
+import math
+import random
+from concurrent.futures import ThreadPoolExecutor
+from io import BytesIO
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+
+MAX_SQUARES = 193_600
+
+EXECUTOR = ThreadPoolExecutor(10)
+
+
+class Splitify(commands.Cog):
+    """Splitifies your avatar!"""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @staticmethod
+    def split_image(img: Image, squares: int) -> list:
```

Comment:
```suggestion
    def split_image(img: Image.Image, squares: int) -> list:
```
Currently you're type hinting to a module.

Reply:
Done.